cmd/notation/blob.go

```diff
@@ -0,0 +1,232 @@
+// Copyright The Notary Project Authors.
```

Comment:
Please move this inside `blob` directory similar to cert, plugin etc. Also, rename file to sign.go.